.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+<!-- Before reporting an issue please check that you are using the latest PHPStan version! -->
+
+### Summary of problem or feature request
+
+<!-- Please describe your problem/feature request here. -->
+
+
+### Code snippet of problem
+
+<!-- Try to reproduce the issue you are facing using https://phpstan.org/ (click "Analyze & Store Result" to get an unique URL) -->
```

Comment:
_a unique_

Reply:
Isn't _an_ unique, as `u` is vowel?

Reply:
It is, but the rule for a/an is not whether the word starts with a vowel or not, but whether it _sounds_ like a vowel or not.
https://www.quora.com/Which-is-the-correct-grammar-usage-a-unique-or-an-unique
https://arstechnica.com/civis/viewtopic.php?t=97084

Reply:
fixed, thanks @Majkl578!